src/client.js

```diff
@@ -98,6 +98,12 @@ function keyFromRecoverySession(session, decryptionKey) {
  *
  * @param {string} opts.userId The user ID for this user.
  *
+ * @param {Object} opts.deviceToImport Device data exported with
+ *     (TODO link to export method) that must be imported to recreate this device.
```

Comment:
(The TODO should be resolved before this gets merged)

Reply:
Done.